FFS-4197: Update Readme for Developer Experience (#1620)
DSACMS/iv-cbv-payroll: README.md

> # Eligibility made easy (Emmy)

Reviewer: nit: Title case?

```suggestion
# Eligibility Made Easy (Emmy)
```

To learn more about the Emmy project, [check out our product page that explains the benefits of Emmy](https://www.cms.gov/medicaid-chip/community-engagement-support/eligibility-made-easy).

Emmy uses consent-based verification (CBV) with multiple data sources, making the process much faster and more efficient than a simple document-upload service. CBV enables additional cost avoidance by optimizing manual document review processes. Rather than having to process incorrect or blurry documents, consent-based verification produces an easily consumed report with essential information. Verification information is returned in a standardized format easy for other systems to process (JSON), allowing integration with existing state systems.

Emmy is under active development by CMS, with new updates released on a 2-week cadence.
## Core Team

> The core team of Emmy is a small group of designers, developers, and strategiestss working for and with the Department of Health and Human Services. We're aided in our effort by members of the open source community.

Reviewer: typo ("strategiestss"), and rewording to be a bit less clunky? wdyt?:

```suggestion
The core team of Emmy is a small group of technologists working at the Department of Health and Human Services. We're aided in our effort by members of the open source community.
```

A list of core team members responsible for the code and documentation in this repository can be found in [COMMUNITY.md](COMMUNITY.md) and [CODEOWNERS.md](./CODEOWNERS.md).

## Documentation

### Repository Structure

- [/.github](./.github) GitHub specific settings files and testing, linting, and CI/CD workflows
- [/app](./app) the Emmy web application built using Rails
- [/bin](./bin) scripts for managing infrastructure
- [/docs](./docs) public documentation for developers
- [/infra](./infra) contains infrastructure-as-code and documentation of replication steps for  Emmy environment
- [/load_testing](./load_testing) load testing resources

See [AGENTS.md](./app/AGENTS.md) for some more information on repo structure.

### Documentation

Public documentation is in the [/docs](./docs/) subfolder.

### Architectural Decision Records

Our ADRs are stored in [CMS Confluence.](https://confluenceent.cms.gov/pages/viewpage.action?pageId=693666588)

## Contributing

> For a guide to running the Emmy project on your own, and submiting changes, see [CONTRIBUTING.md](./CONTRIBUTING.md).

Reviewer: nit: Maybe this is engineer bias but I think putting this link higher in the README would be good so that any technical folks reading the README are more likely to see it. I think it would be fine after the first paragraph (i.e. a new paragraph after "It is currently being piloted for testing and validation purposes.")

Author: agreed, will move up
